acceptance-tests.md

```diff
@@ -36,26 +66,56 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Add a new project**
     * _As a user who is a PI, I should be able to create a project by._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. User has previously been set as a PI.
-        2. User logs in to ColdFront and requests a project by going to Home → Projects and by clicking on ‘Create Project’.
-        3. User requests a resource allocation on the ‘OpenShift’ resource for the above created project.
+        2. User logs in to ColdFront and requests a project by going to Home → Projects and by clicking on `Create Project`. [See the ColdFront documentation here](https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/).
+        3. User requests a resource allocation on the `OpenShift` resource for the above created project.
         4. Administrator approves the request.
         5. Upon approval, a project will be created in OpenShift for this particular allocation. If the user does not already exist in OpenShift the user will be created. The project created will have the following attributes
             1. Project name prefixed by a random 6 char hex
             2. Project ID / namespace as a uuid
             3. Requested quota attributes.
         6. The user will be able to authenticate using Keycloak and their institutional login.
+    * Acceptance tests:
+        1. Setup the user as a PI in ColdFront.
+        2. Log into ColdFront as the user and request a project.
+        3. As the user, request an OpenShift resource allocation in ColdFront.
+        4. Log into ColdFront as an admin and approve the request.
+        5. Validate the project was create with the requested quota:
+            1. For a given project named `012345myproject` for example, check that the given project is listed in the `oc` CLI:
+                1. `oc get Project/012345myproject`<br/> ![Get project](img/acceptance-tests/get-project.png)
```

Comment:
You don't need a `<br/>` tag here. You can just write:

```
1. `oc get Project/012345myproject`

    ![Get project](img/acceptance-tests/get-project.png)
```

But see the earlier comment about replacing these terminal examples with actual text. I would probably format these like this to show both the command and the resulting output:

```
1.
    ```
    $ oc get Project/012345myproject
    NAME             DISPLAY NAME   STATUS
    012345myproject                 Active
    ```
```

Which would render like this:


1.
    ```
    $ oc get Project/012345myproject
    NAME             DISPLAY NAME   STATUS
    012345myproject                 Active
    ```